storyblok/storyblok-js-client
# Hotfix(int-1125): Fix memory leak when resolving relations

## Pull request type

Jira Link: [INT-1125](https://storyblok.atlassian.net/browse/INT-1125)




## How to test this PR

It is possible to use the code below to test resolve relations and check the amount of memory used after the code runs.

```js
import StoryblokClient from "storyblok-js-client";
import { memoryUsage } from "node:process"

const returnMemoryUsage = () => {
  const formatMemoryUsage = (data) => `${Math.round(data / 1024 / 1024 * 100) / 100} MB`

  const memoryData = memoryUsage()

  return {
    rss: `${formatMemoryUsage(memoryData.rss)} -> Resident Set Size - total memory allocated for the process execution`,
    heapTotal: `${formatMemoryUsage(memoryData.heapTotal)} -> total size of the allocated heap`,
    heapUsed: `${formatMemoryUsage(memoryData.heapUsed)} -> actual memory used during the execution`,
    external: `${formatMemoryUsage(memoryData.external)} -> V8 external memory`,
  }
}

const client = new StoryblokClient({
  accessToken: 'TOKEN'
})

console.log('STARTING', returnMemoryUsage())

const res = await client.get('cdn/stories', {
  by_uuids: 'UUIDS',
  version: 'draft',
  resolve_relations: 'RELATIONS',
  language: ''
})

console.log('Finish', returnMemoryUsage())
```

## What is the new behavior?

- Nothing changes in the output

## Other information


[INT-1125]: https://storyblok.atlassian.net/browse/INT-1125?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
